src/hr/algebra/pi/test/TerminalTest.java

```diff
@@ -0,0 +1,69 @@
+package hr.algebra.pi.test;
+
+import hr.algebra.pi.model.Terminal;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+
+public class TerminalTest {
+    Terminal terminal = new Terminal();
+
+    @Test
+    public void testParkSmallCarSuccess() {
```

Comment:
Be careful when unit testing, you usually test one functionality and side-effect, especially if we are talking about 'by the books' examples :)